src/components/access-settings/AccessSettings.tsx

```diff
@@ -434,8 +462,11 @@ const AccessSettings: FunctionComponent<{study: Study}> = ({study}) => {
               }}
               color="primary"
               variant="contained">
-              Save Changes
+              {isCreatingNewMember ? 'Saving...' : 'Save Changes'}
             </Button>
+            <Backdrop open={isCreatingNewMember} sx={{zIndex: theme => theme.zIndex.drawer + 1}}>
+              <CircularProgress />
+            </Backdrop>
```

Comment:
Show a loader while the new member is created

Reply:
nice use of the theme to calculate an appropriate zIndex